ONSdigital/ras-frontstage
# Improve process to update Design System version

# What and why?
This PR updates, improves and documents the process to update the Design System macros in ras-frontstage.

- a `.design-system-version` file has been created where the Design System version can now be set
- the Design System version has been parameterised in the `load-templates` script and that script has been renamed to `load-design-system-templates` to better explain what it does
- the Design System templates have been added to the `.gitignore` file and been removed from the ras-frontstage repo
- the `load-design-system-templates` script has been added to run when the `start` script runs

# How to test?
- remove the Design System macros from your local copy of ras-frontstage (delete the `frontstage/templates/components` and `frontstage/templates/layout` folders)
- run `make start` and see that the Design System macros are loaded and the app runs correctly

# Trello
